packages/clay-shared/src/index.tsx

```diff
@@ -4,7 +4,9 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
+export const noop = () => {};
```

Comment:
I'm a bit hesitant about this, I do not want us to always use this here, if we're not going to end up having some components with dependency on `shared` that is only using the `noop`.